### PR TITLE
chore(broker-redis): lint files

### PR DESCRIPTION
**Pull request by overbit**
Fixes: amplication/amplication#7239

## PR Details

Applies lint rules to all files of the plugin 

## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm build` doesn't throw any error
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: plugins/broker-redis/package.json
Comment (overbit) [inline plugins/broker-redis/package.json:1]: As based on #219, this PR needs to wait for that to be merged before getting merged

File: plugins/broker-redis/src/constants.ts [hunk 1]
```diff
@@ -1,49 +1,58 @@
...
 export const dependencies = {
-    dependencies: {
-        "redis": "3.1.2",
-        "@nestjs/microservices": "8.2.3",
-        "rxjs": "^7.8.1"
-    }
-}
+  dependencies: {
+    redis: "3.1.2",
+    "@nestjs/microservices": "8.2.3",
```
Comment (abrl91) [inline plugins/broker-redis/src/constants.ts:14]: 8.2.3?

File: plugins/broker-redis/webpack.config.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 const webpack = require("webpack");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
-const nodeExternals = require("webpack-node-externals");
 
 /** @type {import("webpack").Configuration} */
 module.exports = {
   mode: "production",
   target: "node",
   entry: "./src/index.ts",
+  externals: ["@amplication/code-gen-utils", "@amplication/code-gen-types"],
```
Comment (abrl91) [inline plugins/broker-redis/webpack.config.js:10]: What does it do?

File: plugins/broker-redis/src/tests/beforeCreateServerPackageJson.spec.ts [hunk 3]
```diff
@@ -3,27 +3,28 @@ import { mock } from "jest-mock-extended";
...
+  it("should add the dependencies required to use Redis to the package.json file", () => {
+    const { updateProperties } = plugin.beforeCreateServerPackageJson(context, {
+      fileContent: "",
+      updateProperties: [{}],
     });
+    expect(updateProperties).toStrictEqual([
+      {
+        dependencies: {
+          redis: "3.1.2",
+          "@nestjs/microservices": "8.2.3",
```
Comment (abrl91) [inline plugins/broker-redis/src/tests/beforeCreateServerPackageJson.spec.ts:24]: Don't we need to change it to v10?

File: plugins/broker-redis/src/constants.ts (hunk 1 above)
Comment (overbit) [reply plugins/broker-redis/src/constants.ts:14]: this PR only run the linting rules and not update the version yet

File: plugins/broker-redis/webpack.config.js (hunk 2 above)
Comment (overbit) [reply plugins/broker-redis/webpack.config.js:10]: it won't bundle these dependencies in the code as they are already installed in DSG

File: plugins/broker-redis/src/tests/beforeCreateServerPackageJson.spec.ts (hunk 3 above)
Comment (overbit) [reply plugins/broker-redis/src/tests/beforeCreateServerPackageJson.spec.ts:24]: Those changes are here:
https://github.com/amplication/plugins/pull/221

**Review by abrl91 — APPROVED** [review]

**Pull request merged**